packages/amplify_auth_cognito/lib/method_channel_auth_cognito.dart

```diff
@@ -277,27 +277,39 @@ class AmplifyAuthCognitoMethodChannel extends AmplifyAuthCognito {
         isSignUpComplete: res["isSignUpComplete"],
         nextStep: AuthNextSignUpStep(
             signUpStep: res["nextStep"]["signUpStep"],
-            codeDeliveryDetails: res["nextStep"]["codeDeliveryDetails"],
+            codeDeliveryDetails: res["nextStep"]["codeDeliveryDetails"] != null
```

Comment:
Nit: Can we extract `res["nextStep"]["codeDeliveryDetails"]` into a variable so that we don't have to keep repeating it? I think it'd help with the readability of the code.